test/controllers/StatusInfoControllerTest.scala

```diff
@@ -14,18 +14,20 @@
  * limitations under the License.
  */
 
-package Controllers
+package controllers
```

Comment:
Can you rename the actual folder rather than just renaming here :) pls and ty.

Reply:
I actually think James did the correct change here 👀 
Am I missing something?